protocols/JabberG/src/jabber_caps.cpp

```diff
@@ -394,7 +394,7 @@ bool CJabberProto::HandleCapsInfoRequest(const TiXmlElement *, CJabberIqInfo *pI
 
 		CJabberClientPartialCaps *pCaps = g_clientCapsManager.GetPartialCaps(JABBER_CAPS_MIRANDA_NODE, m_szFeaturesCrc);
 		if (pCaps) {
-			if (m_bShowOSVersion) {
+			if (m_bShowOSVersion == true) {

```

Comment:
Why that change?

Reply:
@tweimer this to increase code stability.
@ghazan able to not accept this change.

Reply:
I don't see how it increases code stability.

Reply:
Why not?

Reply:
How is it supposed to increase code stability?
It is just unnecessary. 

In general, you better don't change code formatting on pull requests.
You better focus on the actual changes..